### PR TITLE
Add support infrastructure for SDR and FEC

### PR DESCRIPTION
**Pull request by Taylorr82**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,8 +8,8 @@
   "private": false,
   "scripts": {
     "build:env": "sudo apt-get install libzmq3-dev -y && sudo apt install libsocketcan-dev -y && sudo apt install pkg-config -y && sudo apt install python3-dev -y && sudo apt install python3-pip -y && sudo pip3 install numpy",
-    "csp:clone": "sudo rm -rf libcsp && git clone --depth 1 https://github.com/AlbertaSat/libcsp",
-    "csp:configure": "cd libcsp && python3 waf configure --with-os=posix --enable-can-socketcan --enable-rdp --enable-crc32 --enable-hmac --enable-xtea --with-loglevel=debug --enable-debug-timestamp --enable-python3-bindings --with-driver-usart=linux --enable-if-zmqhub --enable-examples",
+    "csp:clone": "sudo rm -rf libcsp && git clone --branch master https://github.com/Albertasat/libcsp.git",
```
Comment (rcunrau) [inline package.json]: I wasn't sure why we had "--depth 1", and I'm not sure why we need "--branch master" either :-)

**Review by rcunrau — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Taylorr82) [reply package.json]: I think the master branch option was a part of the weird depth thing. I'll remove it

**Comment by Taylorr82** [comment]
@rcunrau It says I dismissed your review, not sure what happened. I'm going to merge anyway

**Comment by rcunrau** [comment]
Weird. I was trying to mark the issue I raised as "resolved", so it
wouldn't say I was asking for changes. I probably did it incorrectly, but
you'd still think it would attribute the changes to me :-/

**Comment by Taylorr82** [comment]
I think it happened because I force-pushed this branch after a rebase

**Pull request merged**